nixos/modules/services/misc/weechat.nix

```diff
@@ -46,10 +46,12 @@ in
         Group = "weechat";
         RemainAfterExit = "yes";
       };
-      script = "exec ${pkgs.screen}/bin/screen -Dm -S ${cfg.sessionName} ${cfg.binary}";
+      script = "exec /run/wrappers/bin/screen -Dm -S ${cfg.sessionName} ${cfg.binary}";
```

Comment:
You may want this to be `${config.security.wrapperDir}/bin/screen`?  It appears that a lot of places hard-code the path `/run/wrappers`, so not super-clear that this is a huge deal 😃